website/data/docs-nav-data.json

```diff
@@ -1214,6 +1214,10 @@
                 "title": "Running Vault",
                 "path": "platform/k8s/helm/run"
               },
+              {
+                "title": "Running Vault Enterprise",
```

Comment:
This page name feels a little too much like it's an alternative to the "Running Vault" page, but it's actually more like an addendum. Could something like "Installing Vault Enterprise License" and/or making it a sub-page or a sub-section of "Running Vault" make it a bit clearer how the two pages relate?

Reply:
Good point, I think I'll change the sidebar title to "Enterprise Licensing". I'd kinda like to keep it as a separate page for now for visibility.

Reply:
Updated in 1a453c25bb98fbf996978c949a56bd93f9793bb0